ThirdPartyNotices.txt

```diff
@@ -6,6 +6,7 @@ Do Not Translate or Localize
 This project incorporates components from the projects listed below. The original copyright notices and the licenses under which the .NET Foundation received such components are set forth below. The .NET Foundation reserves all rights not expressly granted herein, whether by implication, estoppel or otherwise.
 
 1.	PedroLamas/DeferredEvents version 1.0.4 (https://github.com/PedroLamas/DeferredEvents)
+2.	FrecherxDachs/UwpNotificationNetCoreTest commit 5c1a4a3 (https://github.com/FrecherxDachs/UwpNotificationNetCoreTest)
```

Comment:
Realizing, it'd be good to link to the file where we're using it as well and for what here. I'll update the DeferredEvents one as a separate PR.